types/nightwatch/index.d.ts

```diff
@@ -5094,16 +5168,17 @@ export interface ElementCommands {
      *   }
      * }
      *
+     * @see https://nightwatchjs.org/api/takeElementScreenshot.html
      */
     takeElementScreenshot(
         selector: Definition,
         callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<string>) => string,
```

Comment:
The return type of the callback should probably be `void` otherwise it will complain that the callback should return a string while the examples above do not return anything.